test/model/binding.dart

```diff
@@ -241,6 +246,12 @@ class FakeFirebaseMessaging extends Fake implements FirebaseMessaging {
   Stream<String> get onTokenRefresh => _tokenController.stream;
 
   StreamController<RemoteMessage> onMessage = StreamController.broadcast();
+
+  /// Controls [TestZulipBinding.firebaseMessagingOnBackgroundMessage].
+  ///
+  /// Calling [StreamController.add] on this will cause a call
+  /// to any handler registered through the latter.
+  StreamController<RemoteMessage> onBackgroundMessage = StreamController.broadcast();
```

Comment:
I'm getting lost about what "the latter" refers to. I'm not sure what the corresponding "former" is. Would it be accurate to replace "the latter" with `[TestZulipBinding.firebaseMessagingOnBackgroundMessage]`? But "the latter" doesn't seem like the right label for that, because nothing is mentioned before it.

Reply:
Hmm yeah. That is indeed what I meant it to refer to — basically I didn't want to repeat that very long name. :-)

Maybe I'll say "registered through that method"; that's at least accurate, it's potentially ambiguous with `StreamController.add`, but hopefully it's clear enough which of the two mentioned methods is meant.